test/test_db_operation.py

```diff
@@ -0,0 +1,43 @@
+
+import unittest
+import sqlite3
+import sys
+sys.path.append('/home/nkwa-tambe/report-sscrap')
```

Comment:
You should remove this. Do you need it?

Reply:
hello sir @stephane-segning ..
sir i need those lbaries sir..

i am using the unittest libary for testing the db operation(testing the "store_analysis_results_in_database" function)

i am using the sqlite3 libary to be able to create a connection with the db/ "store_analysis_results_in_database" function i am testing

i need the sys libary to be able to call the  "store_analysis_results_in_database" function that i need to test as it is not in same directory with my test script.

hence sir, i need them for my test_script to work efficiently.

Reply:
Ok, let's imagine I run it on my machine, the path `/home/nkwa-tambe/report-sscrap` doesn't exist here. How am I going to do?

Reply:
You didn't changed a thing here